indigo_app/static/javascript/indigo/views/work_commencements.js

```diff
@@ -68,6 +90,22 @@
         // we can't iterate over this in-place, so do this instead
         while (form.elements.provisions.selectedOptions.length > 0) form.elements.provisions.selectedOptions[0].selected = false;
       }
+    },
+
+    handleCheckboxesChange(e) {
+      const parentElement = e.target.closest("li");
+      const nestedListElement = parentElement.querySelector('ul');
+      if(nestedListElement) {
+        for(const checkBox of nestedListElement.querySelectorAll("input[name='provisions']")) {
```

Comment:
You can eliminate the if and nestedListElement completely and just iterate directly over the inputs, eliminating two variables which are used only once AND the if statement:

```
for (const checkBox of e.target.closest('li').querySelectorAll("ul input[name='provisions']")) {
```

Also space in `for(` -- for isn't a function call.

Reply:
done